ibm/service/power/data_source_ibm_pi_key.go

```diff
@@ -21,26 +21,30 @@ func DataSourceIBMPIKey() *schema.Resource {
 		ReadContext: dataSourceIBMPIKeyRead,
 		Schema: map[string]*schema.Schema{
 
-			helpers.PIKeyName: {
+			// Arguments
+			Arg_KeyName: {
 				Type:         schema.TypeString,
 				Required:     true,
-				Description:  "SSHKey Name to be used for pvminstances",
+				Description:  "SSH key name for a pcloud tenant",
 				ValidateFunc: validation.NoZeroValues,
 			},
-			helpers.PICloudInstanceId: {
+			Arg_CloudInstanceID: {
 				Type:         schema.TypeString,
 				Required:     true,
 				ValidateFunc: validation.NoZeroValues,
 			},
-			//Computed Attributes
-			PIKeyDate: {
-				Type:     schema.TypeString,
-				Computed: true,
+
+			// Attributes
+			Attr_KeyCreationDate: {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Date of sshkey creation",
```

Comment:
Do to want to "SSH key" instead of "sshkey" string?

Reply:
Are you talking about the description?

Reply:
yes